zosia16/settings/dev.example.py

```diff
@@ -0,0 +1,52 @@
+from .common import *
+
+DEBUG = True
+# Set some predefined key to make development easier
+# SECRET_KEY = '...'
```

Comment:
But we generate random one there, so if you override it here with constant one, you make sure it won't change.